test/views/wallet/ConfigureMobilecoindView/ConfigureMobilecoindView.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { ConfigureMobilecoindView } from '../../../../app/views/wallet';
+import renderSnapshot from '../../../renderSnapshot';
+
+describe('Configure Mobilecoind View', () => {
+  describe('component', () => {
+    describe('render', () => {
```

Comment:
I think there is a breadcrumb link being rendered on the settings views. We should test that click behavior.